assets/src/edit-story/elements/text/index.js

```diff
@@ -42,6 +43,12 @@ export const hasEditMode = true;
 
 export const isMedia = false;
 
+export const resizeRules = {
+  vertical: false,
```

Comment:
Is this true?

Reply:
Yes. This is correct, per UX.

Reply:
Figma files seem to reflect allowing vertical, too. Is this just not updated?